Clicker/View Controllers/LoginViewController.swift

```diff
@@ -25,16 +26,31 @@ class LoginViewController: UIViewController, GIDSignInUIDelegate {
         signInButton.colorScheme = .light
         view.addSubview(signInButton)
         
+        continueButton = UIButton()
+        continueButton.setTitle("Continue", for: .normal)
+        continueButton.addTarget(self, action: #selector(continueAction), for: .touchUpInside)
+        view.addSubview(continueButton)
+        
```

Comment:
Is this the button to fix the moving to home screen after login issue? If so, I think I know how to fix it. We don't need to add an extra button. Theres a didSignIn delegate method that we can insert. Once they log in, we can pop the LoginVC in there.

Reply:
Take a look at PollsCell for an example. I think I have that method in there